tests/test_debugging.py

```diff
@@ -242,12 +242,13 @@ def circuit():
 
         assert result == expected
 
-    @pytest.mark.parametrize("shots", [None, 0, 1, 100])
+    @pytest.mark.parametrize("shots", [None, 1, 1000000])
     def test_different_shots(self, shots):
         """Test that snapshots are returned correctly with different QNode shot values."""
-        dev = qml.device("default.qubit", wires=2)
+        np.random.seed(51)
```

Comment:
Note that a fixed global seed is now automatically set before every test using a pytest fixture.  So we shouldn't need to do this anymore to ensure consistency.

Reply:
Thanks!